modules/tile-converter/scripts/converter.js

```diff
@@ -34,6 +34,9 @@ function printHelp() {
   console.log(
     '--generate-texture [Enable KTX2 textures generation if only one of (JPG, PNG) texture is provided or generate JPG texture if only KTX2 is provided]'
   );
+  console.log(
+    '--generate-bounding-volumes [Will generate obb and mbs bounding volumes from geometry]'
```

Comment:
Nit: Singular vs plural consistency: generate-texture vs generated-bounding-volumes

Reply:
Yes, I see. I think will be better to rename `--generate-texture` - > `--generate-textures`